src/ports/postgres/modules/graph/test/sssp.sql_in

```diff
@@ -280,9 +280,46 @@ INSERT INTO e VALUES
 (ascii('s'), ascii('t'), 6),
 (ascii('s'), ascii('y'), 7);
 
-select graph_sssp('v',
+CREATE TABLE e_grp(src INT, dest INT, weight float8, grp INT);
```

Comment:
I think might be a good idea to add a test case for an actual negative cycle to make sure we didn’t add any regression for detecting negative cycles